ci/scripts/github/checks.sh

```diff
@@ -23,6 +23,8 @@ fetch_base_branch
 update_conda_env
 
 rapids-logger "Configuring CMake"
+git submodule init
+git submodule update
```

Comment:
Why do we use:
```
git submodule update --init --recursive
```
in some places and
```
git submodule init
git submodule update
```
in others? I think I would prefer the single line option: `git submodule update --init --recursive`

Reply:
Good catch, I prefer the single line as well, will fix.